README.md

```diff
@@ -37,6 +37,19 @@ Open up [localhost:3000](http://localhost:3000) and start clicking around.
 
 Consult [sapper.svelte.dev](https://sapper.svelte.dev) for help getting started.
 
+### Running tests
+
+Functional tests are located in the [`cypress/integration`](cypress/integration) folder. Prior to running tests, you'll need to install [cypress](https://www.cypress.io/) globally:
+```bash
+npm i -g cypress
+```
+
+Then you can run the tests using the following commands :
+```bash
+npm t
+npm run cy:run
+npm run cy:open
+```
```

Comment:
@antony, I've taken care of your request changes and added a link to the application specs.

Reply:
:+1:  That still suggests a global install of cypress though. That needs to be fixed.

Reply:
Made suggested changes for [dev dependency instead of global install](https://github.com/sveltejs/sapper-template/pull/194#discussion_r382356496) and [not abbreviated version of `npm install`](https://github.com/sveltejs/sapper-template/pull/194#discussion_r382357474) in separate review.

For `npm test` 3 lines can be combined I think so did not suggest change [in my comment](https://github.com/sveltejs/sapper-template/pull/194#discussion_r382359027) since those are limited to 1 line. 

Did address the  [`cy:run`](https://docs.cypress.io/guides/guides/command-line.html#cypress-run) (runs all tests headlessly in the Electron browser) vs [`cy:open`](https://docs.cypress.io/guides/guides/command-line.html#cypress-open) (opens the Cypress Test Runner in interactive mode). As I wrote there, we may want to document `cy:open` but I think that should be a new paragraph if we do.